src/main/features/claim/views/check-and-send.njk

```diff
@@ -38,57 +38,80 @@
         )
     }}
 
-    {{ row('Email:', draftClaim.defendant.email.address, paths.defendantEmailPage.uri, bold = true) }}
+    {{ row('Email', draftClaim.defendant.email.address, paths.defendantEmailPage.uri, bold = true) }}
     {{ tableEnd() }}
 
     {{ tableStart('claimAmount', 'Claim amount') }}
-    {{ row('Claim amount:', claimAmountTotal.claimAmount | numeral, paths.amountPage.uri, bold = true) }}
+
+    {{ row('Claim amount breakdown', '', paths.amountPage.uri, bottomBorder = false, bold = true) }}
```

Comment:
these check and send content changes should be in a separate PR, they could get merged very quickly